include/wx/osx/clipbrd.h

```diff
@@ -19,6 +19,101 @@
 // wxClipboard
 //-----------------------------------------------------------------------------
 
+class WXDLLIMPEXP_CORE wxOSXDataSourceItem
+{
+public:
+    virtual ~wxOSXDataSourceItem();
+
+    virtual wxDataFormat::NativeFormat AvailableType(CFArrayRef types) const = 0;
+
+    virtual bool GetData( const wxDataFormat& dataFormat, wxMemoryBuffer& target) = 0;
+
+    virtual bool GetData( wxDataFormat::NativeFormat type, wxMemoryBuffer& target) = 0;
+
+    virtual CFDataRef DoGetData(wxDataFormat::NativeFormat type) const = 0;
```

Comment:
I don't fully understand the relationship between `GetData()` and `DoGetData()` but I think it would be more clear if they were called using less similar names, e.g. `DoGetNativeDataRef()` or something like this for this one.

As above, I'm also not sure why does either of them have to be virtual, they seem to have only one implementation...